src/metrics.rs

```diff
@@ -143,6 +187,13 @@ pub struct Metrics {
     metrics_recv: mpsc::Receiver<AggregatedMetricsMessages>,
     commands_recv: mpsc::Receiver<MetricsCommands>,
     samples: AllocRingBuffer<AggregatedSample>,
+    total_counts: MessageCounts,
+    // The number of clients that have ever been started (includes ones that have
+    // subsequently stopped). To get the number of _currently_ active clients,
+    // subtract stopped from started.
+    clients_started: i64,
```

Comment:
are these signed intentionally?

Reply:
Yes, I as subscribe to "all values representing numbers should be signed: do not choose unsigned simply because a particular numbers is not expected to be negative". Happy to change it for consistency or whatever though.